.github/workflows/ensureSCCoreDevApproval.yml

```diff
@@ -17,6 +17,7 @@ jobs:
         env:
           GH_PAT: ${{ secrets.GIT_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CONTINUE: false # makes sure that variable is correctly initialized in all cases
```

Comment:
Why is the `env.CONTINUE` not defined in the top level of the job and not in a step?

Reply:
updated